build.gradle.kts

```diff
@@ -9,19 +9,12 @@ buildscript {
 
     build.loadExtraPropertiesOf(project)
 
-    val kotlinVersion: String by extra
-    dependencies {
-        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
-    }
-
-    repositories {
-        gradlePluginPortal()
-    }
 }
 
 plugins {
     base
     id("org.jetbrains.gradle.plugin.idea-ext") version "0.1"
+    kotlin("jvm") apply false
```

Comment:
:o: I'd move this one line up, just for clarity and to prevent any ordering issue

Reply:
Ordering issue?

Reply:
There shouldn't be any between these two plugins.

Reply:
I'd just move the kotlin language plugin up the IDE one, so the list of plugins order reads nicer.